GsaGH/Components/5_Results/ReactionForceDiagrams.cs

```diff
@@ -221,15 +222,22 @@ private enum DisplayValue {
         _selectedDisplayValue.ToString());
     }
 
-    private static string GetNodeFilters(IGH_DataAccess dataAccess) {
-      string nodeList = string.Empty;
-      var ghNoList = new GH_String();
-      if (dataAccess.GetData(1, ref ghNoList)) {
-        nodeList = GH_Convert.ToString(ghNoList, out string tempNodeList, GH_Conversion.Both) ?
-          tempNodeList : string.Empty;
+    private string GetNodeFilters(IGH_DataAccess dataAccess) {
+      string nodeList = "All";
+      var ghType = new GH_ObjectWrapper();
+      if (dataAccess.GetData(1, ref ghType)) {
+        if (ghType.Value is GsaListGoo listGoo) {
+          if (listGoo.Value.EntityType != Parameters.EntityType.Node) {
+            this.AddRuntimeWarning(
+            "List must be of type Node to apply to node filter");
+          }
+          nodeList = "\"" + listGoo.Value.Name + "\"";
+        } else {
+          GH_Convert.ToString(ghType.Value, out nodeList, GH_Conversion.Both);
+        }
       }
 
-      if (nodeList.ToLower() == "all" || string.IsNullOrEmpty(nodeList)) {
+      if (nodeList.ToLower() == "all" || nodeList == "") {
```

Comment:
why not string.isNullOrEmpty?

Reply:
because it is never null, but we could check instead if empty however `""` seems to be shorter? Do you want me to change?

Reply:
yes pls